### PR TITLE
chore: Rebase release 9765ff6 to develop

### PR DESCRIPTION
**Pull request by insignias**
Issue #, if available:

Description of changes:
1. Rebase release 9765ff6 to develop

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench-core/example/infrastructure/integration-tests/support/complex/datasetHelper.ts [hunk 1]
```diff
@@ -75,4 +75,13 @@ export class DatasetHelper {
       })
     );
   }
+
+  public async getddbRecords(dataSetId: string, endpointId?: string): Promise<Record<string, JSONValue>> {
+    return this._awsSdk.helpers.ddb.getItem({
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/integration-tests/support/complex/datasetHelper.ts:80]:
```suggestion
  public async getddbRecords(awsService: AwsService, dataSetId: string, endpointId?: string): Promise<Record<string, JSONValue>> {
    return awsService.helpers.ddb.getItem({
```

File: workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts [hunk 2]
```diff
@@ -82,6 +83,13 @@ describe('datasets create integration test', () => {
         }
       ]);
     });
+    it('doesnt store the authenticated user in ddb', async () => {
+      const { data } = await adminSession.resources.datasets.create();
+
+      const metadata = await new DatasetHelper().getddbRecords(data.id);
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts:89]:
```suggestion
      const mainAwsService = setup.getMainAwsClient('ExampleDataSetDDBTableName');
      const metadata = await DatasetHelper.getddbRecords(mainAwsService, data.id);
```

File: workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts [hunk 3]
```diff
@@ -124,6 +132,13 @@ describe('datasets create integration test', () => {
           }
         }
       });
+
+      // confirm authenticated user isnt present in ddb
+      const metadata = await new DatasetHelper().getddbRecords(
+        dataset.id,
+        response.data.mountObject.endpointId
+      );
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts:140]:
```suggestion
      const mainAwsService = setup.getMainAwsClient('ExampleDataSetDDBTableName');
      const metadata = await DatasetHelper.getddbRecords(
        mainAwsService,
        dataset.id,
        response.data.mountObject.endpointId
      );
```

**Review by bhudnell — APPROVED** [review]
Made some changes to DatasetHelper to work with develop's changes for BYOB testing.

File: .viperlightignore [hunk 4]
```diff
@@ -43,8 +43,8 @@ workbench-core/example/infrastructure/integration-tests/tests/dynamicAuthorizati
 # Fake email for DataSetsService integration tests
 workbench-core/example/infrastructure/integration-tests/tests/datasets/accessPermissions.test.ts:31
 workbench-core/example/infrastructure/integration-tests/tests/datasets/delete.test.ts:30
-workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts:97
 workbench-core/example/infrastructure/integration-tests/tests/datasets/byob.test.ts:61
+workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts:105
```
Comment (bhudnell) [inline .viperlightignore:47]:
```suggestion
workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts:106
```

**Review by bhudnell — APPROVED** [review]

File: workbench-core/example/infrastructure/integration-tests/support/complex/datasetHelper.ts [hunk 5]
```diff
@@ -75,4 +75,13 @@ export class DatasetHelper {
       })
     );
   }
+
+  public static async getddbRecords(awsService: AwsService, dataSetId: string, endpointId?: string): Promise<Record<string, JSONValue>> {
+    return awsService.helpers.ddb.getItem({
+      key: {
+        pk: buildDynamoDbKey(dataSetId, dataSetPrefix),
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/integration-tests/support/complex/datasetHelper.ts:82]: Need to add `buildDynamoDbKey` import

File: workbench-core/example/infrastructure/integration-tests/support/complex/datasetHelper.ts [hunk 6]
```diff
@@ -75,4 +75,13 @@ export class DatasetHelper {
       })
     );
   }
+
+  public static async getddbRecords(awsService: AwsService, dataSetId: string, endpointId?: string): Promise<Record<string, JSONValue>> {
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/integration-tests/support/complex/datasetHelper.ts:79]: Need to add `JSONValue` import

**Review by SanketD92 — APPROVED** [review]
Merge looks right to me.

**Pull request merged**